cli/src/pcluster/templates/cw_dashboard_builder.py

```diff
@@ -147,7 +152,7 @@ def _add_resources(self):
 
         # Head Node logs add custom metrics if cw_log and metrics are enabled
         if self.config.is_cw_logging_enabled:
-            if self.config.scheduling.scheduler == "slurm":
+            if self.config.scheduling.scheduler == "slurm" and is_region_supported(get_region()):
```

Comment:
Why do we use get_region here instead of _stack_region?

Reply:
Stack region was the Cfn token representing the region of the stack I think?